.gitignore

```diff
@@ -4,3 +4,4 @@
 /coveralls-upload.json
 /phpunit.xml
 /vendor/
+.idea
```

Comment:
Should be in your global gitignore: this artifact comes from your local tooling, not from the library.